.docker/keycloak/users.json

```diff
@@ -0,0 +1,98 @@
+[
```

Comment:
curious how this was initially created - to add more users would I just mock them out here and trust the import/export scripts to manage things from there, seeing as the IDs aren't used during import?

Reply:
You can do it either this way or via UI, just make sure that all the ids under the `roles` field are correct. And if you choose to change via UI don’t forget to `make kc-export-users` afterward.

Reply:
Gotcha, so first time you add new users would mean following the README steps to get into the local KC and set them up for export 😄  makes sense!